src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx

```diff
@@ -37,21 +36,14 @@ class GroupTags extends React.Component {
     this.fetchData();
   }
 
-  componentDidUpdate(prevProps) {
```

Comment:
Why remove this?

Reply:
This didn't apply in the organization version (query is always hardcoded - https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/groupDetails/organization/groupTags.jsx#L11). It was to maintain compatibility with how environments in the project version worked